en/docs/references/data-mapping.md

```diff
@@ -1,49 +1,48 @@
 # Data Mapping
 
 You can use a data mapping statement in Choreo to create variables by visually mapping one or more variables.  The most common use of data mapping in Choreo is to map data from an API call response to JSON format. 
-
 Follow this procedure if you want to add a data mapping statement to your integration or service:
 
 1. Go to the **Statements** tab and select **Data Mapping**. You can see the **Input** and **Output** panes.
 
      ![Data mapping input output windows](../assets/img/references/datamapping/input-output-windows.png){.cInlineImage-full}
    
-2. To add an input variable, follow this procedure:
+2. To add an input variable:
```

Comment:
in the above docs we used, follow this procedure, but here we are not, why?

Reply:
+1. Makes sense to use the same. Seems like I have removed by just following the copy editor's suggestion. Added it back.